src/api/batch/farmlands.md

```diff
@@ -0,0 +1,316 @@
+
+<a name="contact">
+### Contact
+
+Exported Farmlands Contact and [Card][] information used for authentication, correspondence and payment.
+
+{% h4 Required Fields %}
+
+|   Field    |  Type  |                            Description                            |
+| :--------- | :----- | :---------------------------------------------------------------- |
+| externalId | String | Id used for keeping imported Centrapay Contact details up to date |
+
+
+{% h4 Optional Fields %}
+
+|  Field  |         Type         |                                            Description                                             |
+| :------ | :------------------- | :------------------------------------------------------------------------------------------------- |
+| name    | String               | The full name used to address the individual                                                       |
+| mobile  | {% dt PhoneNumber %} | Mobile number used for authentication and correspondence                                           |
+| email   | String               | Email address used for authentication and correspondence                                           |
+| primary | Boolean              | `true` if the Contact is the owner of the Farmlands Account                                        |
+| cards   | Array                | [Card][] list used for payment for the authenticated subject. When empty all Cards will be removed |
+
+<a name="card">
+### Card
+
+Exported Farmlands Credit Card information used for importing and updating of a [Patron Code][]
+
+{% h4 Required Fields %}
+
+|   Field    |        Type        |                                                      Description                                                      |
+| :--------- | :----------------- | :-------------------------------------------------------------------------------------------------------------------- |
+| externalId | String             | Id used for keeping imported [Patron Code][] details up to date                                                       |
+| status     | String             | Current state of the Card. Valid values are "active", "inactive" and "archived"                                       |
+| expiry     | {% dt Timestamp %} | Payments will be accepted until this time. When expressed as `YYYY-MM-DD` the Card will be valid up to 23:59:59 NZST. |
+
+{% h4 Optional Fields %}
+
+|      Field      |  Type  |                              Description                              |
+| :-------------- | :----- | :-------------------------------------------------------------------- |
+| barcode         | String | 9 digit field to display in barcode format                            |
+| farmlandsStatus | String | Private field used in all Farmlands payment transaction notifications |
+
+
+<a name="example">
+## Examples
+
+Full Account, Contact and Card Update
```

Comment:
Just removing the title from this isn't enough. I expect to see a sentence explaining the example.

```suggestion
We match perviously created resources using `externalId`. Records are created or updated based on these IDs and we update details to match. In some cases, fields can be left out to update some fields without specifying the whole set.

An example of a Farmlands Asset complete with balance, contacts and card details.
```